cmd/kubeadm/app/cmd/config.go

```diff
@@ -214,9 +215,59 @@ func NewCmdConfigImages(out io.Writer) *cobra.Command {
 		RunE:  cmdutil.SubCmdRunE("images"),
 	}
 	cmd.AddCommand(NewCmdConfigImagesList(out))
+	cmd.AddCommand(NewCmdConfigImagesPull())
 	return cmd
 }
 
+// NewCmdConfigImagesPull returns the `config images pull` command
+func NewCmdConfigImagesPull() *cobra.Command {
+	cfg := kubeadmapiv1alpha1.MasterConfiguration{}
+	kubeadmapiv1alpha1.SetDefaults_MasterConfiguration(&cfg)
+	var cfgPath, featureGatesString string
+	var err error
+	cmd := &cobra.Command{
+		Use:   "pull",
+		Short: "Pull images used by kubeadm.",
+		Run: func(_ *cobra.Command, _ []string) {
+			cfg.FeatureGates, err = features.NewFeatureGate(&features.InitFeatureGates, featureGatesString)
+			kubeadmutil.CheckErr(err)
+			internalcfg, err := configutil.ConfigFileAndDefaultsToInternalConfig(cfgPath, &cfg)
+			kubeadmutil.CheckErr(err)
+			puller, err := images.NewImagePuller(utilsexec.New(), internalcfg.GetCRISocket())
+			kubeadmutil.CheckErr(err)
+			imagesPull := NewImagesPull(puller, images.GetAllImages(internalcfg))
+			kubeadmutil.CheckErr(imagesPull.PullAll())
+		},
+	}
+	AddImagesCommonConfigFlags(cmd.PersistentFlags(), &cfg, &featureGatesString)
+	return cmd
+}
+
+// ImagesPull is the struct used to hold information relating to image pulling
+type ImagesPull struct {
+	puller images.Puller
+	images []string
+}
+
+// NewImagesPull initializes and returns the `config images pull` command
+func NewImagesPull(puller images.Puller, images []string) *ImagesPull {
+	return &ImagesPull{
+		puller: puller,
+		images: images,
+	}
+}
+
+// PullAll pulls all images that the ImagesPull knows about
+func (ip *ImagesPull) PullAll() error {
+	for _, image := range ip.images {
+		if err := ip.puller.Pull(image); err != nil {
+			return fmt.Errorf("failed to pull image [%v]: %v", image, err)
+		}
+		glog.Infof("[config/images/pull] Pulled %v", image)
```

Comment:
just `[images]` is fine for me here. As `image` is a string, use `%s\n` (don't forget the newline)